firmware/src/rdm.h

```diff
@@ -265,6 +265,14 @@ typedef enum {
 
 
 /**
+ * @brief Open Lighting Project manufacturer PIDs.
+ */
+typedef enum {
+  PID_DEVICE_MODEL = 0x8002,
```

Comment:
I really don't like that part of the standard. I don't know anyone who follows it and it's not even obviously from Table A-3 what the category prefixes are.